test/TestUtil.js

```diff
@@ -136,10 +136,36 @@
         teardownTestDiv(createdObjs.mapDiv);
     }
 
+    /**
+     * Disables output via calls to Ext.Logger-methods.
+     */
+    function disableLogger() {
+        var origLog = Ext.Logger.log;
+        var origError = Ext.Logger.error;
```

Comment:
I don't think `Ext.Logger.error` exists, see https://docs.sencha.com/extjs/6.2.0/classic/Ext.html#property-Logger, maybe `warn`?

Reply:
![image](https://user-images.githubusercontent.com/227934/59427780-c1c86200-8ddb-11e9-84b8-701a8981f33f.png)

It exists and is the only method not calling internally into `log`.

Reply:
Absolutely right, not sure what I've seen :see_no_evil: